### ISSUE TITLE
Commit Mono Not Working In Visual Studio Code

### ISSUE THREAD

**Issue by dajeed** [eigilnikolajsen/commit-mono#70]
Font is not being detected by Visual Studio Code. What I mean by that is that whenever I change the font to Commit Mono, the font does not change.
Steps I took:
1. Installed the latest default font under releases.
2. Restarted Visual Studio Code.
3. Restarted my computer.
4. Uninstalled and reinstalled font and tried both OTF and TTF formats.
5. Issue remains unresolved.

https://github.com/eigilnikolajsen/commit-mono/assets/8452880/fcd9e70a-5e8f-4973-970a-6527975ecfd3

**Comment by eigilnikolajsen** [comment]
Try one of the following, to see if it resolves the problem:

1. Add quotes to the font name like `'CommitMono'`
2. Add these lines to the user settings JSON
```
"editor.fontFamily": "CommitMono",
"editor.fontLigatures": true,
```

**Comment by eigilnikolajsen** [comment]
Many thanks. I will fix this first priority. I might need some help testing when done.

**Comment by dajeed** [comment]
I'll be able to test it out for you unless you want to create a testing branch for testing purposes.
Let me know 😄

**Comment by plttn** [comment]
I can replicate this on my Windows machine as well. 

It _looks_ like an issue with the font family metadata? If I open Control Panel and navigate to fonts, only one of the 4 versions shows up, rather than showing as a font family.

![image](https://github.com/eigilnikolajsen/commit-mono/assets/7369280/27a7e05f-216f-4eb8-a1b7-6b3fcd3c2649)

If I delete the 700 bold italic showing here, 700 bold appears to replace it, and repeating until I've deleted all 4 files.

**Comment by eigilnikolajsen** [comment]
I have located the source of the issue and fixed it. If you download the font from the website it should be fixed. I will update the release on Monday, can't right now.

**Comment by plttn** [comment]
Looks like that mostly worked. 

The family is now working as expected in VSCode, however the only two faces in the family as shown are now 400-regular and 700-italic so I would suspect that any regular italic or unitalicized bold would not show up properly: 

![image](https://github.com/eigilnikolajsen/commit-mono/assets/7369280/1309e19f-d9ff-490c-a63b-8388eb0bfc97)

**Comment by eigilnikolajsen** [comment]
Honestly I don't know about Windows and fonts at this point. I get only the italics in the folder, but when I go to font settings in settings all 4 show up. Even though only the italics are showing up in my folder, the regular and bold still works in VS code. I've checked the naming multiple times but all seems to be identical between the 4 styles. If you have any tips, throw them my way, please.
![image](https://github.com/eigilnikolajsen/commit-mono/assets/71004327/15beddc6-6689-46d2-aeb3-b2381c5c55d3)
![image](https://github.com/eigilnikolajsen/commit-mono/assets/71004327/0e648196-c387-416c-91fa-ecbad11d6e37)

Additionally, @vagreargnatry from #68 gets only italic and bold [in his folder](https://github.com/eigilnikolajsen/commit-mono/issues/68#issuecomment-1817594442), so it seems like a gamble what styles show up haha.

**Comment by plttn** [comment]
Ah yep, my bad, looks like it is actually showing in font settings.

**Comment by eigilnikolajsen** [comment]
I am closing this since it is resolved with [release v1.142](https://github.com/eigilnikolajsen/commit-mono/releases/tag/v1.142).

**Comment by phalla-doll** [comment]
Hi, sorry to bother. The latest version does not seem to work on VS Code again (fallback to the 2nd font). Am I missing something?

![Image](https://github.com/user-attachments/assets/62f5c86a-936a-4886-ac32-c1c4cf65eff7)

![Image](https://github.com/user-attachments/assets/ea408ce4-d784-4e9f-95af-0f14c51417c0)
